index.js

```diff
@@ -130,6 +176,8 @@ type Query {
   answer(id: ID!): Answer
   answers(ids: [ID]!): [Answer]
   option(id: ID!): Option
+  pagesAffectedByDeletion(id: ID!): [Page]
```

Comment:
Why didn't we make this `[Page]!`? Should the arg be called `pageId` to be consistent with the below?

Reply:
I think pageId would be clearer for now. Eventually this would be changed to deal with deletion of sections/answers (potentially for piped answers on other pages), but at that time we can introduce a dedicated Input type. Will re-name for the time being.